content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -69,4 +69,4 @@ Confirm you are using the preferred SLI type for your use case. Datadog supports
 [2]: https://app.datadoghq.com/slo
 [3]: https://app.datadoghq.com/slo/new/monitor
 [4]: https://app.datadoghq.com/monitors#create
-[5]: /monitors/service_level_objectives/event/
+[5]: /monitors/service_level_objectives/metric/
```

Comment:
This doesn't work currently but I am guess this is being worked on?

Reply:
Yeah a part of the docs update